packages/gatsby-plugin-sharp/src/__tests__/index.js

```diff
@@ -319,6 +319,53 @@ describe(`gatsby-plugin-sharp`, () => {
       expect(result).toMatchSnapshot()
     })
   })
+
+  describe(`tracedSVG`, () => {
+    it(`doesn't always run`, async () => {
+      const args = {
+        maxWidth: 100,
+        width: 100,
+        tracedSVG: { color: `#FF0000` },
+      }
+
+      let result = await fixed({
+        file,
+        args,
+      })
+
+      expect(result.tracedSVG).toBeUndefined()
```

Comment:
I'd add a jest mock for the `traceSVG` function (it's already exported) and add an assertion that it's not called. What do you think?

Reply:
Any way i could mock this properly?

thought about

```
const spy = jest.spyOn(rest, `traceSVG`)

    beforeEach(() => {
      spy.mockClear()
    })

    afterAll(() => {
      spy.mockClear()
    })
```

with rest

```
const {
  base64,
  fluid,
  fixed,
  queueImageResizing,
  getImageSize,
  ...rest
} = require(`../`)
```

Reply:
This seems trickier than I initially thought

Guess we should merge and iterate on the tests and the code (to make it easier to test) later